configuration/raspi-config.md

```diff
@@ -201,8 +234,20 @@ Enable/disable the experimental OpenGL Fake KMS desktop graphics driver.
 
 Enable/disable the original legacy non-GL VideoCore desktop graphics driver.
 
+#### Compositor
+
+Enable/Display the xcompmgr composition manager
+
+#### Network Interface Names
+
+Enable or disable predictable network interface names. 
+
+#### Network Proxy Settings
+
+Configure the networks proxy settings. 
+
```

Comment:
The "Advanced Settings" menu also has options for "Boot Order" and "Bootloader version".

Reply:
Hmm, not on mine - might need to do an update.

Reply:
Nope still not there.  Ah, maybe because i am on a Pi3. Will try on a Pi4.

Reply:
Yup.